R/generics.R

```diff
@@ -3,7 +3,6 @@
 ### sufficient for hooking that class up to the `formatters` pagination and rendering
 ### machinery.
 
-## nocov start
```

Comment:
I suspect that this was the cause. It is tested and it is covered, but it was not picked up

Reply:
@shajoezhu @ayogasekaram @edelarua this solved the issue for {formatters} about untested documentation (so we can go for releases faster ;)